code/game/area/areas.dm

```diff
@@ -113,7 +113,7 @@
 		if (!( RA.fire ))
 			RA.set_fire_alarm_effect()
 			for(var/obj/machinery/door/firedoor/D in RA)
-				if(!D.blocked)
+				if(!D.welded)
```

Comment:
See the commit about doors. All obj/machinery/door use the same var/welded instead of having different names.